scripts/st2bootstrap-el7.sh

```diff
@@ -442,8 +504,12 @@ install_st2mistral() {
     sudo yum -y install ${PACKAGE_URL}
   fi
 
+  # Configure database settings
+  sudo crudini --set /etc/mistral/mistral.conf database connection "postgresql://mistral:${ST2_POSTGRESQL_PASSWORD}@localhost/mistral"
```

Comment:
Agreed, I made that change in MongoDB and RabbitMQ, but I just followed what was already there for Mistral.

But yeah, I agree 127.0.0.1 is safer because localhost could cause issues with IPv6, etc.